app/src/ui/tutorial/done.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react'
+
+import { encodePathAsUrl } from '../../lib/path'
+import { Button } from '../lib/button'
+import { Dispatcher } from '../dispatcher'
+import { Repository } from '../../models/repository'
+import { PopupType } from '../../models/popup'
+
+const ClappingHandsImage = encodePathAsUrl(
+  __dirname,
+  'static/admin-mentoring.svg'
+)
+const ExploreImage = encodePathAsUrl(__dirname, 'static/explore.svg')
+const NewRepoImage = encodePathAsUrl(__dirname, 'static/repo-template.svg')
+const FolderImage = encodePathAsUrl(__dirname, 'static/file-directory.svg')
+
+interface ITutorialDoneProps {
+  readonly dispatcher: Dispatcher
+
+  /**
+   * The currently selected repository
+   */
+  readonly repository: Repository
+}
+export class TutorialDone extends React.Component<ITutorialDoneProps, {}> {
+  public render() {
+    return (
+      <div id="no-changes">
```

Comment:
I'm not a fan of us reusing the no-changes styles from the blank slate here. I definitely understand the appeal of doing so given that they're so so similar but if the tutorial from the classic application is any indication none of us will ever test that the tutorial works when doing changes in the blank slate or the BlankslateAction component.

Sadly we don't have all that much time left so I think this is just something we're gonna have to accept for the beta but I would love to see an issue to address this in a follow up PR where we either use the `BlankslateAction` component rather than recreate its DOM structure and potentially introduce a few other high-level components to share styling and logic (like perhaps a `BlankslateActionGroup`?) and untangle the styling for the no changes blank-slate from this.

Reply:
yeah i agree @niik, this isn't ideal. i've opened #8343 to track this.